src/Xamarin.CodeAnalysis/Xamarin.CodeAnalysis.targets

```diff
@@ -1,6 +1,10 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <Import Project="Xamarin.CodeAnalysis.Android.targets" Condition="'$(TargetFrameworkIdentifier)' == 'MonoAndroid'" />
+  <Import Project="Xamarin.CodeAnalysis.MaciOS.targets" Condition="'$(TargetFrameworkIdentifier)' == 'Xamarin.iOS'" />
```

Comment:
We're going to start standardizing the IDE side of tooling around the `Apple` name, rather than anything containing `iOS` in the name. This makes for a nicer name that includes all the different *OS names. So please name this target as `Xamarin.CodeAnalysis.Apple.targets`. Also, I'd simplify the import with a single one with multiple `Or`'ed conditions.